examples/expressions/10_expressions_intro_new.py

```diff
@@ -0,0 +1,201 @@
+"""
+.. _example_expressions_intro:
+
+
+Building powerful machine learning pipelines with the skrub DataOps
+=========================
+
+In this example, we show how we can use skrub's DataOps to build a machine learning
+pipeline that pre-processes data, trains a model, and allows for hyperparameter tuning
+on a simple dataset. We will also show how to save the model, load it back,
+and then use it to make predictions on new data.
+
+"""
+
+# %%
+# We begin by loading the employee salaries dataset, which is a regression dataset
+# that contains information about employees and their current annual salaries.
+# By default, the ``fetch_employee_salaries`` function returns the training set.
+# We will load the test set later, to evaluate our model on unseen data.
+
+from skrub.datasets import fetch_employee_salaries
+
+full_data = fetch_employee_salaries().employee_salaries
+training_data = full_data[:8000]
+unseen_data = full_data[8000:]
+
+# %%
+# We can take a look at the dataset using the `TableReport`.
+# This dataset contains numerical, categorical, and datetime features. The column
+# `current_annual_salary` is the target variable we want to predict.
+#
+
+from skrub import TableReport
+
+TableReport(training_data)
```

Comment:
As we are doing "import skrub" below, could we not do it here, and use "skrub.TableReport" to simplify the code
```suggestion
import skrub

skrub.TableReport(training_data)
```